index.js

```diff
@@ -10,8 +10,7 @@ const { authCheck } = require("./middleware/auth");
 const authRoutes = require("./routes/authroutes");
 const paymentRoutes = require("./middleware/payment");
 const connectDB = require("./config/db");
-var url = require("url");
```

Comment:
why remove this??

Reply:
I think i added it earlier while accessing parameters from url , but now that part is in util so no need for that to be here

Reply:
👍